packages/reactive-element/src/decorators/property.ts

```diff
@@ -71,10 +71,11 @@ const standardProperty = (
 
 const legacyProperty = (
   options: PropertyDeclaration,
-  proto: Object,
+  proto: unknown,
```

Comment:
decorators were the main thing using `Object`, so im not sure this is the right solution. extra pair of eyes would be nice, would this break them for consumers? also specifically the property decorator needs an extra cast this way but the rest don't

Reply:
`Object` is arguably the correct type here. I can't remember why we didn't tighten the type to `{constructor: typeof ReactiveElement}` or something. We can't do that now, but that's what I'd use in the cast below.

Reply:
Yeah, we don't want unknown here, this can't be null, undefined, or a primitive. `object` is also a good candidate for this.